core/App/screens/ProofRequestAttributeDetails.tsx

```diff
@@ -86,8 +94,13 @@ const ProofRequestAttributeDetails: React.FC<ProofRequestAttributeDetailsProps>
     }
 
     retrieveCredentialsForProof(proof)
-      .then((credentials) => {
-        setRetrievedCredentials(credentials)
+      .then((retrievedCredentials) => {
+        if (!retrievedCredentials) {
+          return
+        }
+
+        const attributes = processProofAttributes(proof, retrievedCredentials.proofFormats.indy)
+        setProcessedProofAttributes(attributes)
```

Comment:
Is there a need to set these in state?

Reply:
They're used in the `<Record>` below `fields={[...attributes, ...predicates]}`